MIGRATION.md

```diff
@@ -0,0 +1,89 @@
+# Migrating to the `cloudant-node-sdk` library
+This document is to assist in migrating from the `nodejs-cloudant` (module: `cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`). Compatible with Javascript and TypeScript.
+
+## Initializing the client connection
+There are several ways to create a client connection in `cloudant-node-sdk`:
+1. [Environment variables](https://github.com/IBM/cloudant-node-sdk#authentication-with-environment-variables)
+2. [External configuration file](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration)
+3. [Programmatically](https://github.com/IBM/cloudant-node-sdk#programmatic-authentication)
+
+[See the README](https://github.com/IBM/cloudant-node-sdk#code-examples) for code examples on using environment variables.
+
+## Other differences
+1. For storing credentials in a file, `dotenv` package not needed anymore. Check the new operation [here](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration).
+1. No need to fetch database object in advance to perform operations on Documents. For example, in the case of the document update, this would mean calling first `getDocument` to fetch and after `putDocument` to update.
+1. There are no plugin handling anymore.
+1. Authentication mode can be set [this way](https://github.com/IBM/cloudant-node-sdk/tree/master#authentication).
+1. Retries currently not supported.
```

Comment:
Let's be a bit more specific about the retry feature:
```suggestion
1. The plugin for retrying failed requests is not supported.
```

Reply:
Changed in c706430